src/components/v2/Toggle/index.tsx

```diff
@@ -1,19 +1,24 @@
+/** @jsxImportSource @emotion/react */
 import React from 'react';
 import Switch from '@mui/material/Switch';
+import { SerializedStyles } from '@emotion/react';
+import { SwitchBaseProps } from '@mui/material/internal/SwitchBase';
 import { useStyles } from './styles';
 
 export interface IToggleProps {
-  onChange: (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => void;
+  onChange: SwitchBaseProps['onChange'];
   value: boolean;
+  cssStyles?: SerializedStyles;
 }
 
 const label = { inputProps: { 'aria-label': 'Switch' } };
 
-export const Toggle = ({ onChange, value }: IToggleProps) => {
+export const Toggle = ({ onChange, value, cssStyles }: IToggleProps) => {
   const styles = useStyles();
   return (
     <Switch
       sx={styles}
+      css={cssStyles}
```

Comment:
We could clean this up by moving the sx styles to the css prop styles so styles are applied uniformly